test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -281,11 +281,18 @@ private void wipeCluster() throws IOException {
                 Request request = new Request("GET", "_cat/templates");
                 request.addParameter("h", "name");
                 String templates = EntityUtils.toString(adminClient().performRequest(request).getEntity());
-                for (String template : templates.split("\n")) {
-                    if (isXPackTemplate(template)) continue;
-                    adminClient().performRequest(new Request("DELETE", "_template/" + template));
+                if (false == "".equals(templates)) {
+                    for (String template : templates.split("\n")) {
+                        if (isXPackTemplate(template)) continue;
+                        if ("".equals(templates)) {
```

Comment:
should this be teh singular `template` and not the `list,of,templates`, `templates` ?